node/constants/scrabbleLetters.js

```diff
@@ -0,0 +1,31 @@
+const scrabbleLetters = {
```

Comment:
I like how you're isolating your constants in a separate file and important where used.  As a matter of style, I would name this using the CONSTANT_UPPER_SNAKE_CASE form.